src/components/bookdetails/book-details.css

```diff
@@ -39,37 +47,35 @@ span {
   font-weight: 6;
 }
 
-.middlecolor {
+.book-details__middlecolor {
   color: #dad2c7;
 }
 
-.button {
+.book-details__button {
   background-color: #e3893e;
   transition: background-color 0.2s ease-in-out;
   border-radius: 8px;
-  padding: 12px;
+  padding: 5px;
   width: 100%;
   border: none;
   display: block;
   font-weight: 700;
   font-size: 1.25rem;
 }
 
-.button:hover {
+.book-details__button:hover {
   cursor: pointer;
   background-color: #d2782d;
 }
 
 .book-details__cover {
-  border-style: groove;
-  border-color: #5e5e5e;
   margin: auto;
   display: block;
   width: 50%;
-  padding: 1%;
+  padding: 5%;
```

Comment:
don't use % in padding. You can set `margin: 24px auto` to achieve the same effect